tests/WE2E/test_configs/aqm/config.aqm_grid_AQM_NA13km_suite_GFS_v16.yaml

```diff
@@ -13,27 +13,29 @@ workflow:
   DIAG_TABLE_TMPL_FN: diag_table_aqm.FV3_GFS_v16
```

Comment:
The patterns in this file are generally repeated in the next ~90 files in the `test_configs` directory.

In general, these are the things that happen:
* taskgroups are upgraded to a proper list and moved to the `workflow` section.
* Variables in sections that got the new `execution` and `envars` subsections are adjusted
* execution variables are moved from rocoto task definitions to the execution sections of the default config structure

I tried my best not to change the actual configuration of any of the tests.